app-backend/app/src/main/scala/toolrun/package.scala

```diff
@@ -0,0 +1,7 @@
+package com.azavea.rf
+
+import com.azavea.rf.datamodel.{PaginatedResponse, ToolRun}
+
+package object toolrun extends RfJsonProtocols {
+    implicit val paginatedToolRunFormat = jsonFormat6(PaginatedResponse[ToolRun])
```

Comment:
I wish that error message were more clear. I think maybe providing a `PaginatedResponse` json format for scene query parameters would have saved me explicitly writing an `implicit` for its `ToResponseMarshallable`.